tasks/build.py

```diff
@@ -346,20 +348,94 @@ def download_pr(repo_name, branch_name, pr, arch_job_dir):
     # - 'curl' diff for pull request
     # - 'git apply' diff file
     git_clone_cmd = ' '.join(['git clone', f'https://github.com/{repo_name}', arch_job_dir])
-    clone_output, clone_error, clone_exit_code = run_cmd(git_clone_cmd, "Clone repo", arch_job_dir)
+    log(f'cloning with command {git_clone_cmd}')
+    clone_output, clone_error, clone_exit_code = run_cmd(
+        git_clone_cmd, "Clone repo", arch_job_dir, raise_on_error=False
+        )
+    if clone_exit_code != 0:
+        error_stage = 'git clone'
```

Comment:
Very good. Minor suggestion
```suggestion
        error_stage = ERROR_GIT_CLONE
```

and at the beginning (lines 34-79) add something like
```python
ERROR_GIT_CLONE = 'git clone'
```

Reply:
Done, for every occurrence of `error_stage`.